test/test_quantization.py

```diff
@@ -610,6 +615,37 @@ def checkQuantized(model):
         checkQuantized(model)
 
 
+@unittest.skipIf(
+    not torch.fbgemm_is_cpu_supported(),
+    " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
+    " with instruction set support avx2 or newer.",
+)
+@unittest.skip("temoprarily disable the test - enable after known issues are fixed")
+class GraphModePostTrainingQuantTest(QuantizationTestCase):
+    @_tmp_donotuse_dont_inline_everything
+    def test_single_layer(self):
+        r"""Quantize SingleLayerLinearModel which has one Linear module, make sure it is swapped
+        to nnq.Linear which is the quantized version of the module
+        """
+        # eager mode
+        model_eager = quantize(AnnotatedSingleLayerLinearModel(), test_only_eval_fn,
+                         self.calib_data)
+
+        qconfig_dict = {
+            '': QConfig(
+                activation=torch.jit.script(default_observer()())._c,
```

Comment:
no `_c` please. And you can put this transformation inside the quantize() API instead of asking users to do it (including invoking `torch.jit.script`).

Also, I'd like to highlight it again - we shouldn't rely on quantizers being scriptable. There's nothing really preventing us calling back into python implementation (perf is not critical in these cases I suppose). It's really convenient for the users and would make eager and graph apis even closer.

Reply:
> Also, I'd like to highlight it again - we shouldn't rely on quantizers being scriptable. There's nothing really preventing us calling back into python implementation (perf is not critical in these cases I suppose). It's really convenient for the users and would make eager and graph apis even closer.

do you mean observers?

Reply:
supporting non-scriptable observers will be something we plan to do after lockdown